### PR TITLE
Project Polishing

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR
- Adding a new homepage
- Adding a new user profile page
- Showing filters on mobile (previously hidden)
- Adding search
- Form Validation
# [Check these files only](https://github.com/sudipkundu999/CobraStore-React/pull/15/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-deleted-files=true&show-viewed-files=true)
## Please review the code and functionality and let me know of any bugs! 🚀

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -14,41 +14,50 @@ export const Header = () => {
...
+            <i
+              className="fas fa-moon fa-3x theme-toggle"
+              onClick={() => toggleDarkMode()}
+            />
+          ) : (
+            <i
+              className="fas fa-sun fa-3x theme-toggle"
+              onClick={() => toggleDarkMode()}
+            />
+          )}
```
Comment (iRohitGaur) [inline src/components/Header.jsx]: suggestion: you could have kept just one `<i>` here and just changed the classname

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Good to merge 👍🏻

File: src/components/Header.jsx (hunk 1 above)
Comment (sudipkundu999) [reply src/components/Header.jsx]: Done

**Review by bhushannaikaroskar — APPROVED** [review]
Hey sudip, good file structure and clean code. I have no suggestions as such good to merge

**Pull request merged**